src/Microsoft.AspNetCore.Sockets/Internal/AuthorizeHelper.cs

```diff
@@ -12,14 +14,27 @@ namespace Microsoft.AspNetCore.Sockets.Internal
 {
     public static class AuthorizeHelper
     {
-        public static async Task<bool> AuthorizeAsync(HttpContext context, AuthorizationPolicy policy)
+        public static async Task<bool> AuthorizeAsync(HttpContext context, IEnumerable<string> policies)
```

Comment:
Is `policies` guaranteed to be not null? (asking because policy could have been)